src/plugins/data_source_management/public/components/utils.ts

```diff
@@ -123,11 +123,14 @@ export const fetchDataSourceConnections = async (
   hideLocalCluster: boolean
 ) => {
   try {
-    const directQueryConnectionsPromises = dataSources.map((ds) =>
-      getDirectQueryConnections(ds.id, http!).catch(() => [])
-    );
-    const directQueryConnectionsResult = await Promise.all(directQueryConnectionsPromises);
-    const directQueryConnections = directQueryConnectionsResult.flat();
+    const directQueryConnections: DataSourceTableItem[] = [];
+
+    for await (const datasource of dataSources) {
```

Comment:
call out: are we not changing this from parallel requests to synchronously making requests? Does this impact the experience at all?